target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -256,7 +256,7 @@ ramips_board_detect() {
 	*"JHR-N926R")
 		name="jhr-n926r"
 		;;
-	*"K2P")
+	*"K2P/KE2P")
```

Comment:
Will the content of /proc/cpuinfo actually change or do you require
```
*"K2P"|\
*"KE2P")
```

Reply:
Or just drop this case and use generic board detection instead.

Reply:
Well, you cannot simply drop both, as then the DTS compatible will be used, which in return does not match the Device definition name in Makefile. Since this is all mixed in ramips target anyway, I've recently started a PR to harmonize (https://github.com/openwrt/openwrt/pull/2171) the entire target, which will then also remove ramips.sh completely.

For the current situation, you just have to check what `awk 'BEGIN{FS="[ \t]+:[ \t]"} /machine/ {print $2}' /proc/cpuinfo` returns on you KE2P. If it ends with K2P, leave this unchanged. If it ends with K2EP, stick with my suggestion for the moment.

Reply:
@adrianschmutzler The machine string in /proc/cpuinfo is the model string defined in dts. This commit changed that string to "Phicomm K2P/KE2P" so his approach here is correct.

Reply:
Ah, I was not aware of that, I thought this was coming from manufacturer or similar.
Sorry for the disturbance.